### PR TITLE
Updated in assembly course held in Obihiro

### PR DESCRIPTION
**Pull request by tajitaji551**
An assembly course using eLAB-LAB 445 was held in Obihiro on 21/10/2017. More optimal parameter setting was updated.

### CODE REVIEW

**Comment by magicrub** [comment]
@rmackay9 could you take a quick look at this?

File: Tools/Frame_params/eLAB_LAB445_AC34.param [hunk 1]
```diff
@@ -2,28 +2,69 @@
...
+BATT_VOLT_MULT,10
+BATT_VOLT_PIN,2
 BRD_SAFETYENABLE,0
 CH7_OPT,3
 CH8_OPT,4
 COMPASS_EXTERNAL,1
 COMPASS_USE3,0
 FENCE_ALT_MAX,50
 FENCE_RADIUS,150
+FLTMODE1,0
```
Comment (muramura) [inline Tools/Frame_params/eLAB_LAB445_AC34.param]: It is simple to describe only items different from the default. The other items are the same.

File: Tools/Frame_params/eLAB_LAB445_AC34.param [hunk 2]
```diff
@@ -2,28 +2,69 @@
...
 GND_EFFECT_COMP,1
+GPS_TYPE,2
 LOG_BITMASK,397310
 MOT_THST_HOVER,0.2557016
+MOT_SPIN_ARM,0.1
+MOT_SPIN_MIN,0.12
+RTL_ALT,1500
+SERIAL0_BAUD,115
+SERIAL0_PROTOCOL,1
+SERIAL1_BAUD,38
```
Comment (muramura) [inline Tools/Frame_params/eLAB_LAB445_AC34.param]: My telemetry has a baud rate of 57,600.
https://store.shopping.yahoo.co.jp/elab-store/ld013.html

**Comment by tajisoft** [comment]
@muramura Thank you for your comment. I removed some same parameters as the default.

**Comment by muramura** [comment]
The same setting as the default value of APM: Copter V 3.5.3 (1a 85 c 237) is written.
I think that it is unnecessary to have the same value parameter as the default.
ATC_ACCEL_Y_MAX,27000
ATC_ANG_LIM_TC,1
ATC_ANG_YAW_P,4.5
ATC_ANGLE_BOOST,1
ATC_RAT_PIT_FF,0
ATC_RAT_PIT_FILT,20
ATC_RAT_PIT_IMAX,0.5
ATC_RAT_RLL_FILT,20
ATC_RAT_RLL_IMAX,0.5
ATC_RAT_YAW_I,0.018
ATC_RAT_YAW_P,0.18
ATC_SLEW_YAW,6000
BATT_AMP_PERVOLT,17
BATT_CURR_PIN,3
BATT_VOLT_MULT,10
BATT_VOLT_PIN,2
FS_BATT_MAH,0
FS_THR_ENABLE,1
FS_THR_VALUE,975
MOT_SPIN_ARM,0.1
RTL_ALT,1500
SERIAL0_BAUD,115
SERIAL0_PROTOCOL,1
SERIAL1_PROTOCOL,1
SERIAL2_BAUD,57
SERIAL2_PROTOCOL,1
THR_DZ,100
WP_YAW_BEHAVIOR,2
WPNAV_ACCEL,100
WPNAV_ACCEL_Z,100
WPNAV_LOIT_JERK,1000
WPNAV_LOIT_MAXA,250
WPNAV_LOIT_MINA,25
WPNAV_LOIT_SPEED,500
WPNAV_SPEED,500
WPNAV_SPEED_DN,150
WPNAV_SPEED_UP,250

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request closed without merging**